docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -41,6 +41,22 @@ There you can find links to upgrade notes for other versions too.
 
     </project>
     ```
+    - if there are any changes in the your phing configuration, you'll need to make some customizations
```

Comment:
should we use conditions? maybe upgraders will understand it also without `if`, but maybe not. So OK